.travis/push_maps_yaml

```diff
@@ -9,12 +9,13 @@ git config --global user.name "tripleabuilderbot"
 ## clone the website repo, suppress any output
 git clone --quiet https://${PUSH_TO_WEBSITE_TOKEN}@github.com/triplea-game/triplea-game.github.io.git website 2>&1 > /dev/null
 
-## copy in the new maps.yaml
-cp ./triplea_maps.yaml ./website/_data/
+#Split the yaml file into multiple files
+python3 ./.travis/yaml_splitter.py
```

Comment:
If we keep the yaml_splitter.py script - Input/output args would be of help. Before you could see where the input/output was going, with yaml_splitter the paths are baked in, and you really have to crack open the script to see what is going on.

With input/output args, it would potentially be easier for someone reading this script to better understand dependencies and side effects. ie: `./.travis/yaml_splitter ./triplea_maps.yaml ./website/_maps/`